pyrevolve/genotype/lsystem_body/crossover.py

```diff
@@ -0,0 +1,33 @@
+from typing import List
+
+from pyrevolve.genotype.plasticoding import Plasticoding
+
+from ..lsystem_body.genotype import Genotype
+from ..plasticoding.crossover.crossover import CrossoverConfig
+from ..plasticoding.crossover.standard_crossover import standard_crossover
+from .config import Config
+
+
+def crossover(parents: List[Genotype], config: Config) -> Genotype:
+    pars = Plasticize(parents, config)
+
+    crossover_config = CrossoverConfig(
+        config.crossover_prob, config.plasticoding_config
+    )
+
+    out = Deplasticize(standard_crossover({"plane": 0.03}, pars, crossover_config))
+    return out
+
+
+def Plasticize(parents, config):
```

Comment:
we use lower case for functions names in python. So this would be `plasticize`. As an example you can see the function `standard_crossover` above.